### PR TITLE
add passenger_max_requests option per vhost

### PR DESCRIPTION
**Pull request by pulecp**
Signed-off-by: Pavel Pulec kayn@inuits.eu

### CODE REVIEW

**Comment by pulecp** [comment]
This PR actually extends this one #1510

File: spec/defines/vhost_spec.rb [hunk 1]
```diff
@@ -383,6 +383,7 @@
           'passenger_app_env'           => 'test',
           'passenger_ruby'              => '/usr/bin/ruby1.9.1',
           'passenger_min_instances'     => '1',
+          'passenger_max_requests'      => '1000',
```
Comment (smortex) [inline spec/defines/vhost_spec.rb]: [Upstream](https://www.phusionpassenger.com/library/config/apache/reference/#passengermaxrequests) defaults to 0, not 1000.

Comment (Reamer) [inline spec/defines/vhost_spec.rb]: It's similar to this configuration from apache.
https://httpd.apache.org/docs/2.4/en/mod/mpm_common.html#maxconnectionsperchild
Therefore you should set a non-zero value.

File: README.md [hunk 2]
```diff
@@ -2553,6 +2553,10 @@ Sets [PassengerRuby](https://www.phusionpassenger.com/library/config/apache/refe
 
 Sets [PassengerMinInstances](https://www.phusionpassenger.com/library/config/apache/reference/#passengermininstances), the minimum number of application processes to run.
 
+##### `passenger_max_requests
```
Comment (eputnam) [inline README.md]: needs a closing backtick

**Comment by pulecp** [comment]
@eputnam I synced my repo with upstream and reapplied my changes on top of that so it's mergeable.

**Pull request merged**